Dockerfile

```diff
@@ -3,4 +3,4 @@ ADD build/libs/payments.jar /srv/payments.jar
 WORKDIR /srv
 EXPOSE 8080
 RUN mkdir config
```

Comment:
You [should be using a statically registered client in production](https://github.com/WISVCH/payments/blob/ed74371fa4387ab904beeac52974ce6c464f88b2/src/main/java/ch/wisv/payments/SecurityConfig.java#L178). For development I'd simply change the location where the file is written to.